src/jsutils/__tests__/dedent-test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import dedent from '../dedent';
+
+describe('dedent', () => {
+  it('removes indentation in typical usage', () => {
+    expect(dedent`
+      type Query {
+        me: User
+      }
+      
+      type User {
+        id: ID
+        name: String
+      }
+      `).to.equal(
+      'type Query {\n  me: User\n}\n\n' +
+        'type User {\n  id: ID\n  name: String\n}\n',
+    );
```

Comment:
In the current form, these tests are hard to read.
Would be great to have test string in a separate variable and expected string in [readable form](https://github.com/graphql/graphql-js/blob/master/src/language/__tests__/blockStringValue-test.js#L14-L21), e.g.:
```js
const output = dedent`
  type Query {
    me: User
  }
`;

expect(output).to.equal([
  'type Query {',
  '  me: User',
  '}',
].join('\n'));

Reply:
Yes, that looks a bit nicer, particularly if you're using that style already elsewhere. I refactored it already.